components/post_view/post_list/post_list.jsx

```diff
@@ -205,15 +206,11 @@ export default class PostList extends React.PureComponent {
         return {error};
     }
 
-    markChannelAsReadAndViewed = () => {
-        const currentChannelId = this.props.channelId;
-        const prevChannelId = this.props.prevChannelId;
-
-        // Mark previous and next channel as read
+    markChannelAsReadAndViewed = (channelId) => {
         // Posts are marked as read from here to not cause a race when loading posts
         // marking channel as read and viewed after calling for posts in channel
-        this.props.actions.markChannelAsViewed(currentChannelId, prevChannelId);
-        this.props.actions.markChannelAsRead(currentChannelId, prevChannelId);
```

Comment:
i dont  `prevChannelId` used anymore. Can you remove the reference from this file and also from the connected component and the parent components state etc? Search for `prevChannelId` and you should see references for it.

Reply:
Thank you for pointing it out, @sudheerDev, there was quite a lot of dead code around this!